usr/share/rear/verify/NETFS/default/05_start_required_daemons.sh

```diff
@@ -1,42 +1,82 @@
-			portmap
-			StopIfError "Could not start port mapper [portmap] !"
-		elif has_binary rpcbind; then
-			# rpcbind cannot be called multiple times, so start it only if
-			# it is not yet running
-			rpcinfo -p localhost >&8 2>&1 || rpcbind
-			StopIfError "Could not start port mapper [rpcbind] !"
-		else
-			Error "Could not find any portmapper (tried portmap and rpcbind) !"
-		fi
-
-		# check that portmapper is running
-		# note: on some systems portmap can take a second or two, to be accessible. Hence the loop.
-		max_portmap_checks=5
-		until rpcinfo -p localhost >&8 2>&1; do
-			[ $max_portmap_checks -gt 0 ]
-			StopIfError "portmapper is not running, even though we started it"
-			let max_portmap_checks--
-			sleep 1
-		done
+# use plain 'rpcinfo -p' to check if RPC service is available
+# instead of using 'rpcinfo -p localhost' because the latter
+# does not work on some systems while the former works everywhere
+# see https://github.com/rear/rear/issues/889
+#
+# make it no longer fatal when RPC status rpc.statd is unavailable
+# see https://github.com/rear/rear/issues/870
+#
+# we do not use the progress bar any more
+# therefore all references to FD8 are removed
+# in particular '>&8 2>&1' is replaced by '&>/dev/null'
+# because the progress bar mechanism had swallowed all data
+# see https://github.com/rear/rear/pull/874
+# and https://github.com/rear/rear/issues/887
+#
+# first steps to be prepared for 'set -eu' by
+# using 'command || Error' instead of 'command ; StopIfError'
+# and predefining all used variables
+# see https://github.com/rear/rear/wiki/Coding-Style
+#
+local backup_url_scheme=$( url_scheme "$BACKUP_URL" )
+# nothing to do when backup_url_scheme is not "nfs"
+test "nfs" = "$backup_url_scheme" || return
+# predefine all used variables
+local attempt=""
+local portmapper_program=""
+# the actual work
+LogPrint "Starting required daemons for NFS: RPC portmapper (portmap or rpcbind) and rpc.statd if available."
+# newer Linux distros use rpcbind instead of portmap
+if has_binary portmap ; then
+    portmapper_program="portmap"
+    # just run portmap because portmap can be called multiple times without harm
+    portmap || Error "Starting RPC portmapper '$portmapper_program' failed."
+    LogPrint "Started RPC portmapper '$portmapper_program'".
+elif has_binary rpcbind ; then
+    portmapper_program="rpcbind"
+    # rpcbind cannot be called multiple times
+    # so start it only if it is not yet running
+    rpcinfo -p &>/dev/null || rpcbind || Error "Starting RPC portmapper '$portmapper_program' failed."
+    LogPrint "Started RPC portmapper '$portmapper_program'".
+else
+    Error "Could not find a RPC portmapper program (tried portmap and rpcbind)."
+fi
+# check that RPC portmapper service is available and wait for it as needed
+# on some systems portmap/rpcbind can take some time to be accessible
+# hence 5 attempts each second to check that RPC portmapper service is available
+for attempt in $( seq 5 ) ; do
```

Comment:
I wonder as you use `seq` now if this should move from PROGS to REQUIRED_PROGS `conf/GNU/Linux.conf`?
Or, we could replace it by `$(echo 1 2 3 4 5)` ?